docs/tests/integration/test_logcollector/test_location/index.md

```diff
@@ -0,0 +1,22 @@
+# Test location
+
+## Overview 
+
+These tests check if the introduced configuration works as expected for valid values and if the logcollector 
+response for analyzing requests works properly.
+
+## Objective
+
+Confirm that the different options for logcollector configuration work and are correctly loaded.
```

Comment:
```suggestion
Confirm that the different location option values for logcollector configuration work and are correctly loaded.
```

Reply:
Fixed in 8e8e84a2a739ed54d49446ac6e5db42ccb2d59dd